eng/references.targets

```diff
@@ -1,8 +1,15 @@
 <Project>
   <PropertyGroup>
-    <ContractDependencyPaths>$(RefPath)</ContractDependencyPaths>
+    <ContractDependencyPaths>$(RefPath);@(ReferencePath->'%(RelativeDir)'->Distinct())</ContractDependencyPaths>
+    <AssemblySearchPaths>$(AssemblySearchPaths);$(RefPath)</AssemblySearchPaths>
     <FrameworkPathOverride>$(RefPath)</FrameworkPathOverride>
-    <AssemblySearchPaths>$(AssemblySearchPaths);$(RefPath);{RawFileName}</AssemblySearchPaths>
+    <IncludeDefaultReferences Condition="'$(IncludeDefaultReferences)' == ''">true</IncludeDefaultReferences>
+    <IncludeNETStandardReference Condition="'$(IncludeDefaultReferences)' == 'true' and
```

Comment:
why we are not setting IncludeNETStandardReference to true when TFI is netstandard ?

Reply:
References for .NETStandard are coming via the TargetingPack by default, no need to manually reference netstandard.dll.